draft-ietf-quic-tls.md

```diff
@@ -908,6 +918,80 @@ the connection that is hosted on stream 0.  This sequence number is not visible
 to QUIC.
 
 
+## Packet Number Protection {#pn-encrypt}
+
+QUIC packets are protected using a key that is derived from the current set of
+secrets.  The key derived using the "pn" label is used to protect the packet
+number from casual observation.  The packet number protection algorithm depends
+on the negotiated AEAD.
+
+Packet number protection is applied after packet protection is applied (see
+{{aead}}).  The ciphertext of the packet is sampled and used as input to an
+encryption algorithm.
+
+For packets with a long header, the ciphertext starting
+immediately after the packet number is used (that is, octet 17 onwards).
```

Comment:
Immediately after the packet number is now the payload length; this needs to be updated.

Reply:
+1. This needs to be fixed.